src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef WIN32 || WIN64
+#include "resource.h"
```

Comment:
I've worked with resource files in the past, but don't quite remember the details. Just looking at this, all that happens is a define being included. Why is this necessary?